src/components/TermsandConditionsModal.vue

```diff
@@ -31,7 +26,7 @@
         :text="$t('actions.accept')"
         :alt="$t('actions.accept')"
         variant="solid"
-        class="px-6 p-3"
+        class="px-6 p-3 accept-button"
```

Comment:
There is nothing wrong with this. You can move the other classes in `accept-button` to keep it clean.
```suggestion
        class="accept-button"
```

```css
.accept-button {
  @apply px-6 px-3;
}

@media only screen and (max-width: 1223px) and (orientation: landscape) {
  .accept-button {
    @apply w-full m-2;
  }
}
```